src/insights_client/__init__.py

```diff
@@ -193,11 +193,10 @@ def run_phase(phase, client, validated_eggs):
             continue
         client_debug("Attempting phase '%s' with egg '%s'" % (phase['name'], egg))
 
-        # setup the env
-        insights_env = {
-            "INSIGHTS_PHASE": str(phase['name']),
-            "PYTHONPATH": str(egg)
-        }
+        # prepare the environment
+        pythonpath = os.environ.get("PYTHONPATH", "")  # type: str
+        pythonpath = str(egg) if not pythonpath else "{}:{}".format(egg, pythonpath)
```

Comment:
i'd suggest something that i find slightly more readable:
```python
env_pythonpath = str(egg)
pythonpath = os.environ.get("PYTHONPATH", None)
if pythonpath:
    env_pythonpath += ":" + pythonpath
```
ideally it's a perfect use case for the walrus operator, sadly it's too new for this code (added in 3.8, baseline for the code is 3.6)